cicd-scripts/customize-mco.sh

```diff
@@ -17,6 +17,7 @@ elif [[ "$(uname)" == "Darwin" ]]; then
   SED_COMMAND='sed -i '-e' -e'
 fi
 
+source ./scripts/test-utils.sh
```

Comment:
Did you get any errors due to this missing? I seem to be able to execute the tests fine without.

Reply:
I met error when I ran `make e2e-tests-in-kind` without customized images.